features/step_definitions/framework_application_steps.rb

```diff
@@ -12,7 +12,7 @@
   step "I click a link with text '#{link_title}'"
 end
 
-Then(/^I answer all questions on that page$/) do
+Then(/^I answer all questions on that page and those following until I'm back here$/) do
```

Comment:
Ah yeah I guess so - this was my attempt at making the step name generic to allow reuse, because it actually works for any page. If I do change the name I should probably also change it to explicitly look for the declaration overview in case anyone decides to call it from some different starting page.

What about `I'm back on the original page`?

Reply:
Actually I'm not really sure it's right at all - it doesn't actually answer any questions on _this_ page.